routes.js

```diff
@@ -0,0 +1,82 @@
+var Todo = require('./models/todo')(Todo);
+
+module.exports = (server) => {
+    //Test 1: Create a Todo
+    server.post('/todos', async (req, res, next) => {
+        try {
+            let objects = await Todo.create({
+                title: req.body.title,
+                completed: false
+            });
+            res.status(201);
+            res.charSet('utf-8');
+            res.send(objects.dataValues);            
+        } catch (err) {
+            console.log("There is an error with post/todos : ${err}");
+        }
+    });
+
+    //Test 2: List all Todos
+    server.get('/todos', async (req, res, next) => {
+        try {
+            let objects = await Todo.findAll();
```

Comment:
Hopefully `todoObjects` suffices? `objects` is a pretty dumb name in retrospect hah